components/Storyblok/SbCampaignPage/CampaignHero.tsx

```diff
@@ -80,23 +80,14 @@ export const CampaignHero = ({
         <div className={styles.contentInnerWrapper(isFullWidthImage, heroBgColor)}>
           <FlexBox direction="col">
             {!!logoFilename && (
-              <picture>
-                {logoImageSources.map(({ srcSet, media }, index) => (
-                  <source
-                    key={`logo-${index}`}
-                    srcSet={srcSet}
-                    media={media}
-                  />
-                ))}
-                <img
-                  src={logoImageSources[0].srcSet} // Use the first source as the default image
-                  alt={logoAlt || 'Campaign logo'}
-                  width={originalWidth}
-                  height={originalHeight}
-                  fetchPriority="high"
-                  className={styles.heroLogo(logoAlignment, isFullWidthImage)}
-                />
-              </picture>
+              <img
+                src={processedLogo}
+                alt={logoAlt || 'Campaign logo'}
+                width={originalWidth}
+                height={originalHeight}
+                fetchPriority="high"
+                className={styles.heroLogo(logoAlignment, isFullWidthImage)}
+              />
```

Comment:
Simplify logo image for Campaign Hero - no need to generate srcset as the max width is set as 430px in the wrapper anyway. We processed it to crop it to a width of 430px if the original width is > than 430px
https://deploy-preview-603--adapt-giving.netlify.app/athletics-giving-day